src/OrchardCore/OrchardCore.Email.Abstractions/MailMessageBody.cs

```diff
@@ -0,0 +1,17 @@
+namespace OrchardCore.Email;
+
+/// <summary>
+/// Represents a body for <see cref="MailMessage"/>.
+/// </summary>
+public class MailMessageBody
```

Comment:
Add convenience methods for `HasText()` and `HasHtml()` that use `string.IsNullOrEmpty()`.